scripts/cxg_admin.py

```diff
@@ -104,7 +104,7 @@ def transfer_collections(ctx, curr_owner, new_owner):
 
     with db_session_manager() as session:
         collections = session.query(DbCollection).filter(DbCollection.owner == curr_owner).all()
-
+        new_owner_collections = session.query(DbCollection).filter(DbCollection.owner == new_owner).all()
         if collections is not None:
```

Comment:
Is this ever true, [all](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.all) never returns None. It returns a list.

Reply:
```suggestion
        if collections.all():
```